pkg/receiver.go

```diff
@@ -245,6 +255,91 @@ func (receiver *SsfReceiverImplementation) DeleteReceiver() {
 	}
 }
 
+func (receiver *SsfReceiverImplementation) UpdateStreamStatus(status StreamStatus) (StreamStatus, error) {
```

Comment:
We should perhaps have a method called "EnableStream" and "DisableStream", instead of "UpdateStreamStatus". That might be easier for developers to use.